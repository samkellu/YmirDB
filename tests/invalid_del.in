set a 1 2 3
set b 2 3 4
push a b
list entries
del c
del b
del a
list entries
