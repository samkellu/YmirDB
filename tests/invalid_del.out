> ok

> ok

> ok

> b [2 3 4]
a [b 1 2 3]

> not permitted

> ok

> b [2 3 4]

> 
bye
